hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMContext.java

```diff
@@ -71,6 +74,14 @@ public static SCMContext emptyContext() {
     this.scm = scm;
   }
 
+  public void setDeletedBlockLog(DeletedBlockLog deletedBlockLog) {
```

Comment:
You should not pollute SCMContext with DeletedBlockLog, please find a better place.